### DISCUSSION TITLE
to disable any resource attribute, we use environment variable OTEL_EXPERIMENTAL_RESOURCE_DISABLED_KEYS

### DISCUSSION THREAD

**Discussion by krishnaexplore** [open-telemetry/opentelemetry-java#6334] [Q&A] [answered]
to disable any resource attribute, we use environment variable OTEL_EXPERIMENTAL_RESOURCE_DISABLED_KEYS

is this variable name going to change in near future? what is convention on EXPERIMENTAL variables/features

**Comment by jkwatson** [comment, marked as answer]
yes, if it has EXPERIMENTAL in the name, it will definitely change in the future.
